vtStorAta/vtStorAta.cpp

```diff
@@ -25,7 +25,7 @@ namespace vtStor
 
 }
 
-void vtStorAtaInit(std::shared_ptr<vtStor::cDriveEnumeratorInterface>& DriveEnumerator)
+void vtStorAtaInit(std::unique_ptr<vtStor::cDriveEnumeratorInterface>& DriveEnumerator)
```

Comment:
Hi team, I have changed share_ptr to unique_ptr in parameter of vtStorAtaInit function in vtStorAta.cpp